stacks/README.md

```diff
@@ -0,0 +1,144 @@
+# Okteto Stacks
+
+A developer-first application format for running containers in Kubernetes
+
+## Why do we need Okteto Stacks?
+
+Helm is a powerful technology to pack your applications and deploy to Kubernetes. It has a powerful engine that makes your application definition declarative. You create new objects or modified them, and Helm takes care of upgrading your resources or deleting the ones you don't need anymore.
```

Comment:
When I read this paragraph, my first thought was: This sounds great, why do we need something else?

I think you should  not talk of helm and instead say talk about how cool docker-compose was, how great it is for developers and how we are bringing it to Okteto Cloud and Kubernetes.